pkg/server/plugin/nodeattestor/gcp/iit.go

```diff
@@ -113,6 +154,14 @@ func (p *IITAttestorPlugin) Configure(ctx context.Context, req *spi.ConfigureReq
 		return nil, newError("projectid_whitelist is required")
 	}
 
+	if len(config.AgentSVIDTemplate) > 0 {
+		tmpl, err := template.New("agent-svid").Parse(fmt.Sprintf("%s/%s", svidPrefix, config.AgentSVIDTemplate))
+		if err != nil {
+			return nil, newErrorf("failed to parse agent svid template: %q", config.AgentSVIDTemplate)
+		}
+		p.svidTemplate = tmpl
```

Comment:
`svidTemplate` should either be set under the lock or be a part of the config struct (see other comment)